src/coreclr/jit/ee_il_dll.cpp

```diff
@@ -1147,6 +1177,12 @@ void Compiler::eeAllocUnwindInfo(BYTE*          pHotCode,
                                  CorJitFuncKind funcKind)
 {
 #ifdef DEBUG
+    // Fake splitting currently does not handle unwind info for cold code
+    if (pColdCode && JitConfig.JitFakeProcedureSplitting())
+    {
+        return;
+    }
+
```

Comment:
Same comment here as for reserveUnwindInfo: move to just before the `allocUnwindInfo` call, and add a JITDUMP printout.